.php-cs-fixer.dist.php

```diff
@@ -9,6 +9,8 @@
  * file that was distributed with this source code.
  */
 
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+
```

Comment:
```suggestion
```

We should avoid this as much as we can, as the class is not needed here, and it can only bring weird things 😅